Sources/StreamChat/Controllers/ChannelListController/ChannelListController.swift

```diff
@@ -129,51 +128,14 @@ public class ChatChannelListController: DataController, DelegateCallable, DataSt
     
     override public func synchronize(_ completion: ((_ error: Error?) -> Void)? = nil) {
         startChannelListObserverIfNeeded()
-        setupEventObserversIfNeeded(completion: completion)
-    }
```

Comment:
Is this logic covered by tests? It would be great to capture scenario like:
Given the list controller is created & synchronize is called, only 1 API call is made.

Reply:
Yes, in `ChannelListController_Tests.test_synchronize_callsChannelQueryUpdater`